src/blocks/video.js

```diff
@@ -9,11 +9,11 @@ module.exports = Block.extend({
   // more providers at https://gist.github.com/jeffling/a9629ae28e076785a14f
   providers: {
     vimeo: {
-      regex: /(?:http[s]?:\/\/)?(?:www.)?vimeo.com\/(.+)/,
+      regex: /(?:http[s]?:\/\/)?(?:www.)?vimeo\.co(?:.+(?:\/)([^\/].*)+$)/,
```

Comment:
Mine is also fine with https – `/https?/` matches `http` and `https`. What do you mean by 'less steps'? I updated mine to allow for a missing URL scheme, like the original.